src/components/LogRow/BaseRow/index.tsx

```diff
@@ -209,8 +183,8 @@ const RowContainer = styled.div<{
   ${({ shared }) => shared && `background-color: ${yellow.light3};`}
   ${({ bookmarked }) => bookmarked && `background-color: ${yellow.light3};`}
   ${({ highlighted }) => highlighted && `background-color: ${red.light3};`}
-
-  width: unset !important;
+  
```

Comment:
nit. this doesn't show up right in GitHub's preview haha but there are two random spaces on L186 to delete